Test2/Tests/SettingPageBase.cs

```diff
@@ -0,0 +1,150 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+
+namespace Test2.Tests
+{
+
+    internal class SettingPage
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "https://localhost:5001/Settings";
+        }
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+        private void WaitForReady()
+        {
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            wait.Until(ExpectedConditions.ElementToBeClickable(By.Id("save")));
+        }
+        private void WaitForAlert()
+        {
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.AlertIsPresent());
+        }
+
+        [Test]
+
+        public void CurrencyVer()
+        {
+            List<string> actuale = new List<string>();
+            List<string> expected = new List<string>() { "$ - US dollar", "€ - euro", "£ - Great Britain Pound", "₴ - Ukrainian hryvnia" };
+            IWebElement curren = driver.FindElement(By.Id("currency"));
+            SelectElement currenSeletct = new SelectElement(curren);
+            foreach (IWebElement element in currenSeletct.Options)
+            {
+                actuale.Add(element.Text);
+            }
+            Assert.AreEqual(expected, actuale);
+        }
+
+        [TestCase("dd/MM/yyyy")]
+        [TestCase("dd-MM-yyyy")]
+        [TestCase("MM/dd/yyyy")]
+        [TestCase("MM dd yyyy")]
+
+        public void DateFormat(string format)
```

Comment:
No empty line here, please.
```suggestion
        [TestCase("MM dd yyyy")]
        public void DateFormat(string format)
```